modules/creating-infra-machines.adoc

```diff
@@ -3,10 +3,17 @@
 // * post_installation_configuration/cluster-tasks.adoc
 
 [id="creating-infra-machines_{context}"]
-= Creating infrastructure machines
+= Creating infrastructure machines with a dedicated configuration
 
 If you need infrastructure machines to have dedicated configurations, then you must create an infra pool.
 
+[NOTE]
+====
+You do not need a custom pool unless you change a specific node to have a different set of machine configs applied to it. You do not need a custom machine config pool or machine config for infra nodes to work correctly, as it is not a strict requirement for tasks like node labeling, scheduling specific workloads, isolation, or preventing other workload scheduling.
+
+If you have altered a machine config, like removing a `worker` label from an infra node, you must create a custom pool to properly upgrade your node.
```

Comment:
This sentence doesn't quite make sense? You aren't altering a machine config you're removing a label - are you saying if you remove all labels from a an infra node, you should create a custom pool to ensure it gets upgraded?

Reply:
@kikisdeliveryservice I think the part stating `If you have altered a machine config` is misleading. From the context of this specific BZ I'm trying to solve (and based on my limited understanding), the removal of the `worker` label requires a custom pool to be created to properly upgrade the node.

Is that the only case that would require a custom pool? If so, I could change to something like:
>If you remove the `worker` label from an infra node, you must create a custom pool to properly upgrade your node.

Reply:
Gotcha - yes a node that is managed by the cluster/MCO needs to always be a member of a MCP.